hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -179,6 +179,13 @@
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_RETRY_COUNT("frontend.startup.topic.schema.loading.retry.count", 3),
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_THREAD_POOL_SIZE("frontend.startup.topic.schema.loading.thread.pool.size", 16),
 
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_ENABLED("frontend.publishing.startup.validation.enabled", false),
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_TOPIC_NAME("frontend.publishing.startup.validation.topic.name", "pl.allegro.tech.skylab.test"),
```

Comment:
Use something more generic not connected to Allegro internals :)

Reply:
good catch ;) 9f49a03